atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -176,200 +153,67 @@ private SweepResults doRun(TableReference tableRef,
         //     we encounter
         long sweepTs = sweeper.getSweepTimestampSupplier().getSweepTimestamp(
                 unreadableTimestampSupplier, immutableTimestampSupplier);
-
-        byte[] startRow = MoreObjects.firstNonNull(nullableStartRow, PtBytes.EMPTY_BYTE_ARRAY);
-        RangeRequest range = RangeRequest.builder()
+        CandidateCellForSweepingRequest request = ImmutableCandidateCellForSweepingRequest.builder()
```

Comment:
Yes. That would be nice to change but this is currently how the progress table works.